csrc/executor_utils.cpp

```diff
@@ -50,11 +50,306 @@
 #include <nvfuser_resources/warp.h>
 #include <nvfuser_resources/welford.h>
 
+#include <nvrtc.h>
+#include <sys/wait.h>
+#include <unistd.h>
```

Comment:
Looks like you added some linux-specific thing, could you refactor your code and move system related things to this file? https://github.com/NVIDIA/Fuser/blob/main/csrc/sys_utils.cpp

Reply:
Okay I will. It's a "blob" of code I wrote separately and dropped it in nvFuser to test. Then `sys_utils` does not support windows so it will still be a limitation